docs/iris/src/whatsnew/contributions_1.11/bugfix_2016-Oct-06_is_circular.txt

```diff
@@ -0,0 +1,2 @@
+- Fixed a bug where the :func:`iris.util._is_circular` would return false where 
```

Comment:
It should render fine with either, but if you mix the styles you get a warning from sphinx but the output is likely correct (although in some circumstances the intent may be misinterpreted. I'd advocate using strictly 1 style only.